dev/db2/11.5/start.sh

```diff
@@ -27,3 +27,7 @@ if [ ! "$(sudo docker ps -q -f name=db2server)" ]; then
 	done
   echo "Local DB2-11.5 instance is ready for Sequelize tests."
```

Comment:
Wait, this is double now

Reply:
Woops!
I removed the one that is not always printed, so the message is also displayed if the DB was already running